web/pgadmin/tools/sqleditor/static/js/components/sections/StatusBar.jsx

```diff
@@ -1,115 +1,171 @@
-
```

Comment:
Even this file shows it is completely changed. Check your PR yourself after raising.

Reply:
I use to check the diff using gitk mostly and it has the option to show diff with just color, which shows me the actual diff only.
I'm not sure why this file behaving like this, checking for that.